connectors/src/api/webhooks/webhook_google_drive.ts

```diff
@@ -39,7 +41,15 @@ const _webhookGoogleDriveAPIHandler = async (
   const workflowRes = await launchGoogleDriveIncrementalSyncWorkflow(
     webhook.connectorId.toString()
   );
+
   if (workflowRes.isErr()) {
+    if (workflowRes.error instanceof RateLimitError) {
+      logger.info(
+        {},
```

Comment:
Can we trace back the workflow in that log ?

Reply:
I added the connectorId and workflowId.